ci-operator/jobs/openshift/cluster-network-operator/openshift-cluster-network-operator-master-presubmits.yaml

```diff
@@ -672,6 +672,94 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-ipi,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    cluster: api.ci
+    context: ci/prow/e2e-openstack
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-cluster-network-operator-master-e2e-openstack
```

Comment:
nit: should we rename this to e2e-openstack-ovn ?

Reply:
do you know the naming rules? I tried just changing this name and the CI said this is the only valid name.

Reply:
@JacobTanenbaum : sorry, didn't know about the naming rules, I was just giving a glance at the other test names and suggested it. Let's leave it like that then.

Reply:
It'd be really cool to know if there's any naming policy rules. For OpenStack there is, and we had to change them a few times.

Reply:
ideally names are auto-generated and we don't have to care about them ;)